### PR TITLE
devops(invitations): refactor to model lib

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

**Comment by KvnMrs** [comment]
The test ' unit-test-commits ' don't pass, i think it look like another error than a code error.  I tryed to fix it but didn't find a solution

**Comment by phpgeekfr** [comment]
> The test ' unit-test-commits ' don't pass, i think it look like another error than a code error. I tryed to fix it but didn't find a solution

look at `libs/invitation/src/lib/+state/invitation.service.spec.ts`

File: libs/invitation/src/lib/+state/invitation.model.ts [hunk 1]
```diff
@@ -1,9 +1,6 @@
-import { InvitationBase } from './invitation.firestore';
-import { Organization, PublicUser, Movie } from '@blockframes/model';
+import { Organization, PublicUser, Movie, InvitationBase } from '@blockframes/model';
 import { Event } from '@blockframes/event/+state/event.model';
 
-export { InvitationStatus, createInvitation } from './invitation.firestore';
-
 export type Invitation = InvitationBase<Date>;
 
 export interface InvitationDetailed extends Invitation {
```
Comment (phpgeekfr) [inline libs/invitation/src/lib/+state/invitation.model.ts:6]: this file is supposed to be removed

File: libs/invitation/src/lib/+state/invitation.service.ts [hunk 2]
```diff
@@ -3,10 +3,8 @@ import { AngularFireFunctions } from '@angular/fire/functions';
 import { CollectionConfig, CollectionService, AtomicWrite } from 'akita-ng-fire';
 import { OrganizationService } from '@blockframes/organization/+state';
 import { AuthService } from '@blockframes/auth/+state';
-import { createPublicUser, PublicUser, User, createPublicOrganization, Organization } from '@blockframes/model';
+import { createPublicUser, PublicUser, User, createPublicOrganization, Organization, createInvitation, Invitation, InvitationDocument } from '@blockframes/model';
```
Comment (phpgeekfr) [inline libs/invitation/src/lib/+state/invitation.service.ts:6]:
```suggestion
import {
  createPublicUser,
  PublicUser,
  User,
  createPublicOrganization,
  Organization,
  createInvitation,
  Invitation,
  InvitationDocument
} from '@blockframes/model';
```

File: libs/organization/src/lib/organization/components/member-request/member-request.component.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { Component, Input, Output, EventEmitter, ChangeDetectionStrategy } from '@angular/core';
 import { PublicUser } from '@blockframes/model';
-import { Invitation } from '@blockframes/invitation/+state/invitation.model';
+import { Invitation } from '@blockframes/model';
```
Comment (phpgeekfr) [inline libs/organization/src/lib/organization/components/member-request/member-request.component.ts:3]: merge the two imports

**Review by KvnMrs — APPROVED** [review]

**Pull request merged**